services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -138,6 +138,24 @@ export const EnvironmentListItem: React.FC<EnvironmentListItemProps> = ({
         }
         return otherRelease?.sourceCommitId + ': ' + otherRelease?.sourceMessage;
     };
+    const getDeploymentAuthor = (): string => {
+        if (!application) {
+            return ``;
+        }
+        if (!(application && application.version === release.version)) {
+            return '';
+        }
+        if (application.deploymentMetaData === null) {
+            return '';
+        }
+        var deployedUNIX = application.deploymentMetaData?.deployTime ?? '';
+        if (deployedUNIX === '') {
+            return 'Deployed by: ' + application.deploymentMetaData?.deployAuthor;
+        }
```

Comment:
e.g. like this:
```suggestion
        const deployedUNIX = application.deploymentMetaData?.deployTime;
        if (deployedUNIX) {
            return 'Deployed by: ' + application.deploymentMetaData?.deployAuthor;
        }
```

Reply:
Without implicitly checking on line 151 if the time exists, line 155 will throw an error when trying to create deployedDate with the error: `'deployedUNIX' is possibly 'undefined'`

Reply:
I'm fine with how the code is written now.